src/main/java/fr/inria/diversify/mutant/pit/GradlePitTaskAndOptions.java

```diff
@@ -0,0 +1,58 @@
+package fr.inria.diversify.mutant.pit;
+
+/**
+ * Created by Daniele Gagliardi
+ * daniele.gagliardi@eng.it
+ * on 21/07/17.
+ */
+
+public class GradlePitTaskAndOptions {
+
+    public static boolean descartesMode = false;
+
+    public static boolean evosuiteMode = false;
+
+    public static final String OPT_WITH_HISTORY = "enableDefaultIncrementalAnalysis = ";
+
+    public static final String OPT_VALUE_REPORT_DIR = "reportDir = 'build/pit-reports'";
+
+    public static final String OPT_TARGET_CLASSES = "targetClasses = ";
+
+    public static final String OPT_VALUE_FORMAT = "outputFormats = ['CSV','HTML']";
+
+    public static final String OPT_TARGET_TESTS = "targetTests = ";
+
+    public static final String PROPERTY_ADDITIONAL_CP_ELEMENTS = "additionalMutableCodePaths = ";
+
+    public static final String PROPERTY_EXCLUDED_CLASSES = "excludedClasses = ";
+
+    //Apparently Unsupported by PIT plugin
+    public static final String OPT_ADDITIONAL_CP_ELEMENTS = "classPath = ";
```

Comment:
I think that options for the plugins is timeoutConstInMillis and not timeoutConst.
In the same way of the jvmArgs you must specify whole the argument and not only the value:
jvmArgs = ['-Xms16G'] and not only '16G'.

In the other hand, I'm asking myself if we have to put those values directly in the code. What do you think about them?